extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -41,6 +41,12 @@ class AmpFitText extends AMP.BaseElement {
     /** @private {number} */
     this.maxFontSize_ = -1;
 
+    /** @private {!Array<unlistenDef>} */
+    this.unlisteners_ = [];
+
+    /** @private {ResizeObserver} */
+    this.observer_ = null;
```

Comment:
There is no need to store the observer, also unlistener doesn't need to be an array. I recommend
```
this.resizeObserverUnlistener_ = null;
```
Then in `layoutCallback` and `unlayoutCallback` you check the `resizeObserverUnlistener_`